examples/platform/realtek/dac_provider/CommonDACProvider.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
```

Comment:
```suggestion
 *    Copyright (c) 2025 Project CHIP Authors
```
Applies to other newly added files as well